dashboards-reports/package.json

```diff
@@ -72,6 +72,9 @@
     "css-what": "^5.0.1",
     "ansi-regex": "5.0.1",
     "json-schema": "0.4.0",
-    "ws": "^7.4.6"
+    "ws": "^7.4.6",
+    "minimatch": "^3.0.5",
+    "terser": "^4.8.1",
+    "moment": "^2.29.4"
```

Comment:
Were these used before? Where are they coming from?

Reply:
Yes, all are existing dependencies. Just updating the version.

Reply:
@rupal-bq can you please check if they are also present in OpenSearch-Dashboards? If so, we need not add the dependencies here again.

Reply:
In Dashboards this is the current state in 2.x:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.x/package.json
```
    "minimatch": "^3.0.4",
    "moment": "^2.24.0",

Reply:
fyi, in OSD, there's a PR pending to update `minimatch` to `3.0.5`, merged in main, pending approval for `2.x`
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2654/

Reply:
Thanks @ps48, I didn't know that. Some are present in OpenSearch-Dashboards. I'll update PR accordingly.

Reply:
does dashboards resolving it to 3.0.5 prevent plugins resolving it to 3.0.4? I think Dashboards package.json direct dependency will not affect transitive dependencies?